docker_entrypoint.sh

```diff
@@ -40,189 +62,304 @@ ADV_TOR_MAXSOCKSCONNS=$(yq e '.advanced.tor.maxsocksconns' ${BITMONERO_DIR}/star
+ MASKED="true"
+ echo '  RPC Username:' >> ${BITMONERO_DIR}/start9/stats.yaml
  echo '    type: string' >> ${BITMONERO_DIR}/start9/stats.yaml
  echo '    value: "'"$RPC_USERNAME"'"' >> ${BITMONERO_DIR}/start9/stats.yaml
- echo '    description: Username for connecting to the unrestricted Monero RPC' >> ${BITMONERO_DIR}/start9/stats.yaml
+ echo "    description: Username for connecting to the Monero daemon's unrestricted RPC Interface" >> ${BITMONERO_DIR}/start9/stats.yaml
  echo '    copyable: true' >> ${BITMONERO_DIR}/start9/stats.yaml
  echo '    masked: false' >> ${BITMONERO_DIR}/start9/stats.yaml
  echo '    qr: false' >> ${BITMONERO_DIR}/start9/stats.yaml
- echo '  Unrestricted RPC Password:' >> ${BITMONERO_DIR}/start9/stats.yaml
+ echo '  RPC Password:' >> ${BITMONERO_DIR}/start9/stats.yaml
  echo '    type: string' >> ${BITMONERO_DIR}/start9/stats.yaml
  echo '    value: "'"$RPC_PASSWORD"'"' >> ${BITMONERO_DIR}/start9/stats.yaml
- echo '    description: Password for connecting to the unrestricted Monero RPC' >> ${BITMONERO_DIR}/start9/stats.yaml
+ echo "    description: Password for connecting to the Monero daemon's unrestricted RPC Interface" >> ${BITMONERO_DIR}/start9/stats.yaml
  echo '    copyable: true' >> ${BITMONERO_DIR}/start9/stats.yaml
  echo '    masked: true' >> ${BITMONERO_DIR}/start9/stats.yaml
  echo '    qr: false' >> ${BITMONERO_DIR}/start9/stats.yaml
- RPC_USER_PASS="${RPC_USERNAME}:${RPC_PASSWORD}@"
 else
  RPC_USER_PASS=""
+ MASKED="false"
 fi
-echo '  Unrestricted RPC Connection String (LAN):' >> ${BITMONERO_DIR}/start9/stats.yaml
+echo '  Restricted RPC URL (LAN):' >> ${BITMONERO_DIR}/start9/stats.yaml
 echo '    type: string' >> ${BITMONERO_DIR}/start9/stats.yaml
-echo '    value: "'"https://$RPC_USER_PASS$RPC_LAN_ADDRESS:$STARTOS_PROXY_PORT"'"' >> ${BITMONERO_DIR}/start9/stats.yaml
+echo '    value: "'"https://$RPC_USER_PASS$RPC_LAN_ADDRESS_RESTRICTED:$STARTOS_REVPROXY_PORT"'"' >> ${BITMONERO_DIR}/start9/stats.yaml
+echo "    description: Connection string for accessing the Monero daemon's RPC over LAN" >> ${BITMONERO_DIR}/start9/stats.yaml
+echo '    copyable: true' >> ${BITMONERO_DIR}/start9/stats.yaml
+echo '    masked: '$MASKED >> ${BITMONERO_DIR}/start9/stats.yaml
+echo '    qr: true' >> ${BITMONERO_DIR}/start9/stats.yaml
+echo '  Restricted RPC URL (Tor):' >> ${BITMONERO_DIR}/start9/stats.yaml
+echo '    type: string' >> ${BITMONERO_DIR}/start9/stats.yaml
+echo '    value: "'"https://$RPC_USER_PASS$RPC_TOR_ADDRESS_RESTRICTED:$MONERO_RPC_PORT_RESTRICTED"'"' >> ${BITMONERO_DIR}/start9/stats.yaml
+echo '    description: Connection string for accessing the Monero RPC over Tor' >> ${BITMONERO_DIR}/start9/stats.yaml
+echo '    copyable: true' >> ${BITMONERO_DIR}/start9/stats.yaml
+echo '    masked: '$MASKED >> ${BITMONERO_DIR}/start9/stats.yaml
+echo '    qr: true' >> ${BITMONERO_DIR}/start9/stats.yaml
+echo '  Unrestricted RPC URL (LAN):' >> ${BITMONERO_DIR}/start9/stats.yaml
```

Comment:
```suggestion
echo '  Unrestricted RPC URL (Localhost):' >> ${BITMONERO_DIR}/start9/stats.yaml
```

Reply:
now reads "Unrestricted RPC URL (Internal)" so as to be consistent with other labeling